build-llvm.py

```diff
@@ -88,6 +95,27 @@ def parse_parameters(root_folder):
 
                         """),
                         action="store_true")
+    parser.add_argument("--build-type",
```

Comment:
Since we know possible values, should we be setting a list of choices here to avoid invalid input?

Reply:
We could. I don't think I chose to because I didn't want to keep track of any new options that may be added in the future and I assume that if people want to use this option, they know what they are doing.

Reply:
I mean, we _are_ tracking a known good revision which will get updated more often than build types. Current set of build types seems to satisfy all CBL-related use cases and as new ones pop-up, users can raise an issue in the repository to have them included.

Reply:
Very true plus release is really what people will be using anyways because of the speed up. I'll clean this up, thanks for the review.